tests/assets/plugins/blockifiers/async_blockifier.py

```diff
@@ -0,0 +1,61 @@
+"""Async CSV Blockifier - Steamship Plugin."""
+from typing import Union
+
+from assets.plugins.blockifiers.csv_blockifier import CsvBlockifier
+
+from steamship import SteamshipError
+from steamship.app import Response, create_handler
+from steamship.base import Task, TaskState
+from steamship.base.binary_utils import to_b64
+from steamship.plugin.blockifier import Blockifier
+from steamship.plugin.inputs.raw_data_plugin_input import RawDataPluginInput
+from steamship.plugin.outputs.block_and_tag_plugin_output import BlockAndTagPluginOutput
+from steamship.plugin.service import PluginRequest
+
+STATUS_CHECK_KEY = "bbq-time"
+STATUS_MESSAGE = "Still working!"
+
+
+class AsyncCsvBlockifier(CsvBlockifier, Blockifier):
```

Comment:
This demonstrates how to make a blockifier async! 🚢 🚢 🚢 🚢

Reply:
🚢 Beautiful!